Index: app/src/main/java/com/example/android_cjj118/ui/video/detail/VideoDetailFragment.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/src/main/java/com/example/android_cjj118/ui/video/detail/VideoDetailFragment.java b/app/src/main/java/com/example/android_cjj118/ui/video/detail/VideoDetailFragment.java
--- a/app/src/main/java/com/example/android_cjj118/ui/video/detail/VideoDetailFragment.java	
+++ b/app/src/main/java/com/example/android_cjj118/ui/video/detail/VideoDetailFragment.java	(date 1714718221162)
@@ -33,7 +33,7 @@
 
     private boolean isPlay;
     private boolean isPause;
-    private String url = "http://7xjmzj.com1.z0.glb.clouddn.com/20171026175005_JObCxCE2.mp4";
+    private String url = "https://vt1.doubanio.com/202405031404/d532d64c84b7a80fb68b0f511a1f9517/view/movie/M/703140793.mp4";
 
     private OrientationUtils orientationUtils;
     private GSYVideoOptionBuilder gsyVideoOption;
